cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -296,19 +294,16 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             }
         }
 
-        if (evaluatorRun.violations.isNotEmpty()) {
-            println("Rule violations found.")
-            throw ProgramResult(2)
-        }
-    }
-
-    private fun printSummary(errors: List<RuleViolation>) {
-        val counts = errors.groupingBy { it.severity }.eachCount()
+        val counts = evaluatorRun.violations.groupingBy { it.severity }.eachCount()
 
         val errorCount = counts[Severity.ERROR] ?: 0
         val warningCount = counts[Severity.WARNING] ?: 0
         val hintCount = counts[Severity.HINT] ?: 0
 
-        println("Found $errorCount errors, $warningCount warnings, $hintCount hints.")
+        if (errorCount > 0 || warningCount > 0) {
+            println("Found $hintCount hints only.")
+            throw ProgramResult(2)
+        }
+        println("Found: $errorCount errors, $warningCount warnings, $hintCount hints.")
```

Comment:
When I said "keep the exact wording" I really meant it 😉 Please drop the ":" after "Found". And add a blank line before this one. Oh, and the `println()`s are swapped 😉

Reply:
I'm apparently not fully awake, yet... Should be better now.